mlflow/tracking/client.py

```diff
@@ -353,15 +353,17 @@ def search_runs(self, experiment_ids, filter_string="", run_view_type=ViewType.A
     # Registered Model Methods
 
     @experimental
-    def create_registered_model(self, name):
+    def create_registered_model(self, name, tags=None):
         """
         Create a new registered model in backend store.
 
         :param name: Name of the new model. This is expected to be unique in the backend store.
+        :param tags: A list of :py:class:`mlflow.entities.model_registry.RegisteredModelTag`
```

Comment:
For run tags, client-level APIs accept a dictionary of tags and converts them to a list of `RunTag` objects before passing them to the backend store (see https://github.com/mlflow/mlflow/blob/master/mlflow/tracking/_tracking_service/client.py#L88). I think we should do the same thing here.